### PR TITLE
Actually async ucr export

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://manage.dimagi.com/default.asp?273850#1479752

When we were exporting UCR reports, we were generating the tables synchronously. This moves the generation of the actual report content into the async task.

### CODE REVIEW

File: corehq/apps/userreports/reports/util.py [hunk 1]
```diff
@@ -26,3 +36,72 @@ def has_location_filter(view_fn, *args, **kwargs):
         getattr(filter_, 'location_filter', False)
         for filter_ in report.ui_filters
     )
+
+
+class ReportExport(object):
```
Comment (proteusvacuum) [inline corehq/apps/userreports/reports/util.py]: I wasn't sure where best to put this, so I put it in `util`. It looks like the convention in this module (`userreports.reports`) is to create a new file per object. Should I put this in a `report_object.py` file instead?

**Review by emord — APPROVED** [review]
This looks ok for now, but definitely has the feel of "bug that needed get to fixed quickly" that the rest of our report code has.

Not saying that that is your fault at all though. I don't currently have a better suggestion than what you've done. I'll add this PR to the ucr backlog to make sure that it won't get dropped

**Comment by proteusvacuum** [comment]
@emord fair enough. I'm curious how you would have gone about doing this differently as an interrupt ticket.

**Comment by emord** [comment]
> I'm curious how you would have gone about doing this differently as an interrupt ticket.

Offlined that I don't actually have an idea about how to do this better at the moment. just thought that maybe I should think through it some more at a later time

**Pull request merged**